### DISCUSSION TITLE
deployment/fandq

### DISCUSSION THREAD

**Discussion by giscus** [LiteyukiStudio/LiteyukiBot#29] [Announcements]
# deployment/fandq

常见问题 设备上Python环境太乱了，pip和python不对应怎么办？ 请使用/path/to/python -m pip install -r requirements.txt来安装依赖， 然后用/path/to/python main.py来启动Bot， 其中/path/to/python是你要用来运行Bot的可执行文件 为什么我启动后机器人没...

https://bot.liteyuki.icu/deployment/fandq.html

**Comment by Liteyuki** [comment]
test
